test/remote/gateways/remote_plexo_test.rb

```diff
@@ -0,0 +1,151 @@
+
+  def test_successful_purchase
+    response = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success response
+    assert_equal 'You have been mocked.', response.message
+  end
+
+  def test_failed_purchase
+    response = @gateway.purchase(@amount, @declined_card, @options)
+    assert_failure response
+    assert_equal 'denied', response.params['status']
+    assert_equal '96', response.error_code
+    assert_equal 'You have been mocked.', response.message
+  end
+
+  def test_successful_authorize_and_capture
+    auth = @gateway.authorize(@amount, @credit_card, @options)
+    assert_success auth
+
+    assert capture = @gateway.capture(@amount, auth.authorization)
+    assert_success capture
+    assert_equal 'You have been mocked.', capture.message
+  end
+
+  def test_failed_authorize
+    response = @gateway.authorize(@amount, @declined_card, @options)
+    assert_failure response
+    assert_equal '96', response.error_code
+    assert_equal 'denied', response.params['status']
+    assert_equal 'You have been mocked.', response.message
+  end
+
+  def test_partial_capture
+    auth = @gateway.authorize(@amount, @credit_card, @options)
+    assert_success auth
+
+    assert capture = @gateway.capture(@amount - 1, auth.authorization)
+    assert_success capture
+  end
+
+  def test_failed_capture
+    response = @gateway.capture(@amount, '123')
+    assert_failure response
+    assert_equal 'An internal error occurred. Contact support.', response.message
+  end
+
+  def test_successful_refund
+    purchase = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success purchase
+
+    assert refund = @gateway.refund(@amount, purchase.authorization, @cancel_options)
+    assert_success refund
+    assert_equal 'You have been mocked.', refund.message
+  end
+
+  def test_partial_refund
+    purchase = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success purchase
+
+    assert refund = @gateway.refund(@amount - 1, purchase.authorization, @cancel_options.merge({ type: 'partial-refund' }))
+    assert_success refund
+  end
+
+  def test_failed_refund
+    response = @gateway.refund(@amount, '123', @cancel_options)
+    assert_failure response
+    assert_equal 'An internal error occurred. Contact support.', response.message
+  end
+
+  def test_successful_void
+    auth = @gateway.authorize(@amount, @credit_card, @options)
+    assert_success auth
+
+    assert void = @gateway.void(auth.authorization, @cancel_options)
+    assert_success void
+    assert_equal 'You have been mocked.', void.message
+  end
+
+  def test_failed_void
+    response = @gateway.void('123', @cancel_options)
+    assert_failure response
+    assert_equal 'An internal error occurred. Contact support.', response.message
+  end
+
+  def test_successful_verify
+    response = @gateway.verify(@credit_card, @options.merge(@cancel_options))
+    assert_success response
+    assert_match %r{You have been mocked.}, response.message
```

Comment:
Is there a reason to switch from `assert_equal` to `assert_match` here? I think either would be fine, but am wondering why there are some of each.

Reply:
You're right, I have changed it to `assert_equal` to keep it consistent with other tests as there is no any specific reason to use that.